src/main/java/io/github/bbortt/event/planner/domain/Responsibility.java

```diff
@@ -46,6 +46,10 @@
     @Column(name = "name", length = 50, nullable = false)
     private String name;
 
+    @Size(min = 4, max = 23)
+    @Column(name = "color", length = 23)
```

Comment:
Warum `max = 23`?
Wenn man die Farben immer in Hex angibt ist max = 7

Reply:
es ist schwierig, hex könnte glaube ich auch 4 zeichen lang sein sein (e.g. `#FC0` = `#FFCC00`). und man könnte eben auch andere werte speichern. der Scheduler akzeptiert dann einfach alle in css möglichen farben. z.b. den literalen farbwert `red`. oder rgba: das wäre dann (mit allen lücken etc) 23 Zeichen max.

Reply:
um die eingabe also serverseitig nicht mehr validieren zu müssen (regex der hex akzeptiert) habe ich das feld einfach so eingerichtet, dass alles reinpasst.. :(